Doc/library/collections.rst

```diff
@@ -359,7 +359,7 @@ Common patterns for working with :class:`Counter` objects::
     dict(c)                         # convert to a regular dictionary
     c.items()                       # convert to a list of (elem, cnt) pairs
     Counter(dict(list_of_pairs))    # convert from a list of (elem, cnt) pairs
-    c.most_common()[:-n-1:-1]       # n least common elements
+    c.most_common()[-n:]            # n least common elements
```

Comment:
But, this is not the same thing:

```python
>>> from collections import Counter
>>> Counter('aaabbc').most_common()[-2:]
[('b', 2), ('c', 1)]

>>> Counter('aaabbc').most_common()[:-2-1:-1]
[('c', 1), ('b', 2)]
```

Notice that ordering of the new one is different.

Reply:
The current code has the intended behavior which is to order starting from the most extreme value (lowest count).  This matches the behavior of `heapq.nsmallest()`.


```
>>> from collections import Counter
>>> c = Counter(a=5, b=3, c=8, d=2, e=9)
>>> n = 3
>>> c.most_common()[:-n-1:-1]
[('d', 2), ('b', 3), ('a', 5)]

>>> from heapq import nsmallest
>>> counts = [5, 3, 8, 2, 9]
>>> nsmallest(n, counts)
[2, 3, 5]
```

Reply:
I see where I went wrong now.😅 
Thanks for clarifying.